internal/interpreter/ffi.rs

```diff
@@ -249,23 +207,25 @@ pub unsafe extern "C" fn slint_interpreter_struct_destructor(val: *mut StructOpa
 }
 
 #[no_mangle]
-pub extern "C" fn slint_interpreter_struct_get_field<'a>(
-    stru: &'a StructOpaque,
+pub extern "C" fn slint_interpreter_struct_get_field(
+    stru: &StructOpaque,
     name: Slice<u8>,
-) -> Option<&'a ValueOpaque> {
-    stru.as_struct()
-        .get_field(std::str::from_utf8(&name).unwrap())
-        .and_then(|v| unsafe { (v as *const Value as *const ValueOpaque).as_ref() })
+) -> *mut Value {
```

Comment:
(could be `Option<Box<Value>>`, but i don't know if cbingen would generate the right stuff)

Reply:
I tried that as well, but unfortunately it's only available for C. (See the box test in cbindgen repo)